docs/03_Get_Started/00_Get_started_with_ESP32.md

```diff
@@ -72,11 +64,12 @@ It is the entry-point of your firmware and contains the instruction to perform t
 #define B_LED_PIN GPIO_NUM_32
 #define Y_LED_PIN GPIO_NUM_33
 
-
-void led_blinker_task () {
-    gpio_pad_select_gpio(B_LED_PIN);
+void led_blinker_task ()
```

Comment:
if `led_blinker_task` doesn't accept arguments, it should be defined as  `void led_blinker_task(void)`